### ISSUE TITLE
Spindle does not stop on finish

### ISSUE THREAD

**Issue by 9a4gl** [McNugget6750/pcb-gcode#4]
Using CNC2418 with GRBL 0.9 and Candle 1.0.11

After everything is finished spindle keeps spinning. With original pcb-gcode pcb-gcode-3.6.2.4 spindle stooped after everything has finished.

**Comment by McNugget6750** [comment]
You're right. I believe my spindle turns off on command M4 and most spindles turn off on command M5. This has been an issue with other software for me as well and I keep changing it. Let me try and point you to the right location in the python script.

**Comment by McNugget6750** [comment]
I believe you are using my config file. Therefore you need to go to [profiles/grbl-NuggetMill.pp](https://github.com/McNugget6750/pcb-gcode/blob/master/profiles/grbl-NuggetMill.pp) and check line 60:
`string SPINDLE_OFF    = "M4" + EOL;`
needs to be changed to 

`string SPINDLE_OFF    = "M5" + EOL;`
for most CNC machines. Try that and let me know if that helps!

**Comment by 9a4gl** [comment]
I just found this

 M03 | M-Code | Spindle ON (CW Rotation)
 M04 | M-Code | Spindle ON (CCW Rotation) 
 M05 | M-Code | Spindle Stop

Fast switching rotation with M04 may cause also damage.

**Comment by McNugget6750** [comment]
Certainly. I have my own CNC machine design and for reasons I don't remember the relay output only turns on or off on M3 and M4 respectively.
I should change that at some point as it's not standard.

**Comment by 9a4gl** [comment]
I can confirm that changing to M5 fixes the problem. IMO, if this really can be different on some machines it should go to settings and default should me M5
